utils/models.py

```
@@ -0,0 +1,178 @@
+    pipeline,
+)
+
+
+class Leaning(Enum):
+    LEFT = "left"
+    CENTER = "center"
+    RIGHT = "right"
+
+
+class Model(ABC):
+    def __init__(
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)